packages/xcoobee-cookie-kit-react/src/CookieKitPopup.js

```diff
@@ -223,19 +253,23 @@ export default class CookieKitPopup extends React.PureComponent {
               <button
                 type="button"
                 className={`xb-cookie-kit__button ${BLOCK}__language-picker`}
-                onClick={() => this.setState({ isShown: !isShown })}
+                onClick={e => this.handleLocaleSelectToggle(e)}
               >
                 { selectedLocale }
               </button>
               { countryCode && (
-                <div className={`${BLOCK}__block ${BLOCK}__block--sm`}>
+                <button
+                  type="button"
+                  className={`xb-cookie-kit__button ${BLOCK}__block ${BLOCK}__block--sm ${BLOCK}__country-picker`}
```

Comment:
By moving the `__block` and `__block--sm` to the button we are probably breaking backwards compatibility which is now a concern since this code has been publically published.

Can you see if we can get the styling we want with keeping the `div` but embed the `button` within it?

Reply:
Added some additional styles, so it's ok.